pkg/vtpm/Dockerfile

```diff
@@ -11,6 +11,8 @@
 #    and the server
 
 #Build TPM2-TSS and TPM2-TOOLS
+#### FIX ME !!! this is a temporary image.
+FROM shahzededa/eve-dom0-ztools:latest as dom0
```

Comment:
I will update this after #3986 is merged and published.

Reply:
Go ahead

Reply:
done.